Makefile

```diff
@@ -8,7 +8,7 @@ ORCHESTRATOR = docker
 
 .PHONY: help
 help:
-	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort -k 1,1 | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' Makefile | sort -k 1,1 | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
```

Comment:
With multiple Makefiles you'd end up with something like this:

![image](https://github.com/user-attachments/assets/eac335d0-1c59-4516-a2aa-bd63b62d3be8)

Reply:
Ah because it would list the filename. In that case you should fix the grep call instead. From `man grep`:

```
  -h, --no-filename
         Suppress  the  prefixing  of file names on output.  This is the default when there is
         only one file (or only standard input) to search.

```